### ISSUE TITLE
ESbuild / Vite.js for build

### ISSUE THREAD

**Issue by cheatsnake** [metarhia/BestPractice#6]
ESbuild / Vite.js were not mentioned in the row about builders, although they are already often used (especially in new projects).

**Comment by tshemsedinov** [comment]
Мы пока не использовали, как доберемся до фронта на курсе по ноде, то сравним

**Comment by VladyslavSpasibozhko** [comment]
I agree with @cheatsnake. Vite much faster then Webpack and can be used with Vue, React. In new projects Vite is preferred way to build project instead of using Webpack.

**Comment by letavocado** [comment]
@cheatsnake, @VladyslavSpasibozhko 

 ESbuild can't [top level await](https://github.com/evanw/esbuild/issues/253#issuecomment-657277794) 
 Vite.js used for frontend
 
 Feel free to correct me. :smile: 
 
 Cheers!

**Comment by Pakistanka** [comment]
@VladyslavSpasibozhko Подскажите, в итоге удалось как-то vite и lucid сконфигурировать? Я не могу решить top-level ошибку и еще несколько

**Comment by letavocado** [comment]
@Pakistanka 
top-level await в vite
https://github.com/mozilla/pdf.js/issues/17245#issuecomment-1812360307
```js
{
  build: {
    target: "es2022"
  },
  esbuild: {
    target: "es2022"
  },
  optimizeDeps:{
    esbuildOptions: {
      target: "es2022",
    }
  }
}
```
